jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamSchemasResultSet.java

```diff
@@ -15,22 +15,100 @@
 package software.amazon.timestream.jdbc;
 
 import com.amazonaws.services.timestreamquery.model.ColumnInfo;
```

Comment:
For changes this file, I mostly just swapped the contents of this file with `TimestreamDatabasesResultSet.java` and made it return database names that match the schemaPattern